src/components/header/menu/menuWindow/menuWindow.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import MenuWindowElement from './../menuWindowElement';
+import config from './../../../../config';
+import './menuWindow.css';
+
+function MenuWindow({onClose}) {
+
+    let config1 = config.menu1;
+    let config2 = config.menu2;
+
+    const divMenuWindow = document.querySelector('.menuWindow');
+
+    let config1Length = config1.length;
+    let config2Length = config2.length;
+    let reverseArrNum = config1.length - 1;
+
+    const result = [];
+    for (let i = 0; i < config1Length; i++) {
+        result.push(<MenuWindowElement name={config.menu1[i][0]} link={config.menu1[i][1]} positionY={config.menu1[i][2]}/>)
```

Comment:
Please change config to be an array of objects(instead of array of arrays) - so that access is by property(instead of by index)